<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisioForge Media Monitoring Tool LITE (QT)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Broadcast dump</string>
     </attribute>
     <widget class="QPushButton" name="btAddBroadcastFile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add file</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="edBroadcasts">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>251</height>
       </rect>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ad samples</string>
     </attribute>
     <widget class="QPlainTextEdit" name="edAdSamples">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>251</height>
       </rect>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btAddAdFile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add file</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="edResults">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>401</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btStart">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>691</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;That's a very simplified version, .Net version uses the same SDK but have more features in demo.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
